デプロイ失敗対応

@claude

以下のワークフローエラー解消して

> github-pages-demo@0.1.0 build
> next build

[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   ▲ Next.js 16.0.4 (Turbopack)

   Creating an optimized production build ...
[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
 ✓ Compiled successfully in 3.1s
   Running TypeScript ...
Failed to compile.

./app/components/household/Analysis.tsx:44:25
Type error: Type '{ category: string; amount: number; percentage: number; }[]' is not assignable to type '{ category: CategoryType; amount: number; percentage: number; }[]'.
  Type '{ category: string; amount: number; percentage: number; }' is not assignable to type '{ category: CategoryType; amount: number; percentage: number; }'.
    Types of property 'category' are incompatible.
      Type 'string' is not assignable to type 'CategoryType'.

  42 |           {expenseSummary.length > 0 ? (
  43 |             <>
> 44 |               <PieChart data={expenseSummary} />
     |                         ^
  45 |               <div className="mt-6">
  46 |                 <table className="w-full">
  47 |                   <thead className="border-b border-gray-700">
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.

--- a/app/components/household/Analysis.tsx
+++ b/app/components/household/Analysis.tsx
@@ -1,6 +1,6 @@
 'use client';
 
-import type { Transaction } from '@/app/types/household';
+import type { CategorySummary } from '@/app/types/household';
 import { formatCurrency } from '@/app/utils/date';
 import { getCategoryConfig } from '@/app/constants/categories';
 import PieChart from './PieChart';
@@ -11,7 +11,7 @@ interface AnalysisProps {
   getCategorySummary: (
     month: string,
     type: 'income' | 'expense'
-  ) => Array<{ category: string; amount: number; percentage: number }>;
+  ) => CategorySummary[];
   getMonthlySummary: (month: string) => {
     income: number;
     expense: number;

